projects/js-packages/licensing/components/activation-screen/utils.js

```diff
@@ -64,12 +70,36 @@ export function getProductGroup( productId ) {
 	);
 }
 
+/**
+ * Check if the Social plugin is active.
+ *
+ * @returns {boolean} -- True or False.
+ */
+export function isSocialPluginActive() {
+	return (
+		isPluginActive( JETPACK_SOCIAL_PLUGIN_FILE_NAME ) ||
+		isPluginActive( JETPACK_SOCIAL_PLUGIN_DEV_FILE_NAME ) ||
+		isPluginActive( JETPACK_SOCIAL_PLUGIN_LOCAL_FILE_NAME )
+	);
+}
+
+/**
+ * Check if the Jetpack plugin is active.
+ *
+ * @returns {boolean} -- True or False.
+ */
+export function isJetpackPluginActive() {
+	return (
+		isPluginActive( JETPACK_PLUGIN_FILE_NAME ) || isPluginActive( JETPACK_PLUGIN_DEV_FILE_NAME )
+	);
+}
+
 /**
  * Check if a plugin is active.
  *
- * @param {string} plugin -- The name of the plugin such as jetpack/jetpack.php, social/jetpack-social.php, backup/jetpack-backup.php.
+ * @param {string} plugin -- Plugin filename.
  * @returns {boolean} -- True or False.
  */
-export function isPluginActive( plugin ) {
+function isPluginActive( plugin ) {
```

Comment:
If we're in the My Jetpack context (and it correctly identifies that the development versions of plugins are active), I think we might not need this. There is a `useProduct` hook which should give us [information about the product](https://github.com/Automattic/jetpack/blob/trunk/projects/packages/my-jetpack/_inc/components/connected-product-card/index.jsx#L28-L35), including whether it's active.